src/main/java/Duke.java

```diff
@@ -1,10 +1,50 @@
+import java.util.Scanner;
+
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        //Generates a greeting
+        greeting();
```

Comment:
function name greeting should be imperative such as "greet" instead of "greeting"